.github/workflows/model_registry_test.yaml

```diff
@@ -80,3 +88,9 @@ jobs:
         echo "Dry-run KF Model Registry REST API..."
         export KF_TOKEN="$(kubectl -n default create token default)"
         curl -v -H "Authorization: Bearer "$KF_TOKEN http://localhost:8080/api/model_registry/v1alpha3/registered_models
+
+    - name: Dry-run KF Model Registry REST API UI
+      run: |
+        echo "Dry-run KF Model Registry REST API..."
+        export KF_TOKEN="$(kubectl -n default create token default)"
```

Comment:
Thank you for the PR. All model registry pods should live in the Kubeflow namespace.

Reply:
We should even have a negative test that verifies that it fails with no token or the wrong token

Reply:
Ok, so we should test first with a valid token (default-editor) and the with an invalid token? And wouldn't be that an issue with istio?

Reply:
Here you can see an example of verifying that not anyone can access stuff with an illegal token https://github.com/kubeflow/manifests/blob/88c67229aa0c875a8dbe252f21a11380ab97f5d6/.github/workflows/notebook_controller_m2m_test.yaml#L73

Reply:
Ok, I got what you suggested, we were using a healtcheck for smoke test that's not checking the headers. I've just tested it with a proper endpoint, I've added both test for UI as you suggested and it worked in my local tests.

cc @tarilabs maybe you wanna take a look at this.